package/gluon-core/luasrc/usr/lib/lua/gluon/util.lua

```diff
@@ -188,4 +190,71 @@ function M.log(message, verbose)
 	posix_syslog.syslog(posix_syslog.LOG_INFO, message)
 end
 
+M.subprocess = {}
+
+M.subprocess.DEVNULL = -1
+M.subprocess.INHERIT = nil
```

Comment:
Assigning `nil` to an unset table key is meaningless. Let's just scrap the name "INHERIT" and just always use `nil`.

Reply:
resolved in b5655410e5fb463795b42c8e6372ff840cad7406